src/angular/schematics/ng-add/migrations/secondary-entry-points-migration.ts

```diff
@@ -97,7 +97,10 @@ export class SecondaryEntryPointsMigration extends Migration<null, DevkitContext
 
     // Determine the subpackage each symbol in the namedBinding comes from.
     for (const element of declaration.importClause.namedBindings.elements) {
-      const elementName = element.propertyName ? element.propertyName : element.name;
+      let elementName = element.propertyName ? element.propertyName : element.name;
+      if (CLASS_NAME_RENAMES.has(elementName.text)) {
```

Comment:
Can you explain why this is necessary?

Reply:
Otherwise `import { SbbFormFieldModule } from '@sbb-esta/angular-public/form-field'` will be migrated to `import { SbbInputModule } from '@sbb-esta/angular/form-field'`, which will not work.